aesd-char-driver/aesd-circular-buffer.c

```diff
@@ -106,9 +111,13 @@ char * aesd_circular_buffer_add_entry(struct aesd_circular_buffer *buffer, const
     if(buffer->in_offs == buffer->out_offs) {
         buffer->full = true;
     }
+
     return ret_ptr;
 }
-
+unsigned int update_total_size(struct aesd_circular_buffer *buffer) 
```

Comment:
@sachininja `get_total_size` would probably be a better name here since it doesn't update anything.